common/changes/@microsoft/api-extractor/excerpt-tokens_2022-04-05-16-46.json

```diff
@@ -0,0 +1,10 @@
+{
+  "changes": [
+    {
+      "packageName": "@microsoft/api-extractor",
+      "comment": "Fix an issue where .api.json excerpt text included extra whitespace (GitHub #3316)",
```

Comment:
I adjusted the change log message to focus on the [end user impact](https://rushjs.io/pages/best_practices/change_logs/) instead of implementation details

Reply:
Makes perfect sense. thank you